www/docs/server/validators.md

```diff
@@ -363,6 +363,32 @@ export const appRouter = t.router({
 export type AppRouter = typeof appRouter;
 ```
 
+### With [runtypes](https://github.com/pelotom/runtypes)
+
+```ts twoslash
+import * as t from 'runtypes';
+import { initTRPC } from '@trpc/server';
+
+export const t = initTRPC.create();
+
+const publicProcedure = t.procedure;
+
+export const appRouter = t.router({
+  hello: publicProcedure
+    .input(t.Record({ name: t.String }))
+    .output(t.Record({ greeting: t.String }))
+    .query(({ input }) => {
+      //      ^?
+      return {
+        greeting: `hello ${input.name}`,
+      };
+    }),
+});
+
+export type AppRouter = typeof appRouter;
+```
```

Comment:
Thank you for your contribution. We also have a paired tests file under ./packages/tests (apologies, away from PC so can't recall the exact file) which we like to add libraries to, this way we don't accidentally break compatibility in the future.

Could you please add it there too? I'll be happy to merge then 🙂